spring-core/pom.xml

```diff
@@ -13,7 +13,7 @@
   <description>Togglz - Spring Framework integration</description>
 
   <properties>
-    <spring.version>3.0.7.RELEASE</spring.version>
+    <spring.version>3.1.4.RELEASE</spring.version>
```

Comment:
The `Environment` class was introduced in v3.1. I've imported 3.1.4 as this was the latest fix version of 3.1.